tests/conftest.py

```diff
@@ -0,0 +1,14 @@
+# tests/conftest.py
+import pytest
+
+
+def pytest_addoption(parser):
+    parser.addoption("--dirpath", action="store", default=None, help="Full directory path to create")
```

Comment:
```suggestion
    parser.addoption("--dirname", action="store", default=None, help="Directory name to create")
```
full directory path can be generated dynamically , dependent on OS type